Sources/Turf/Feature.swift

```diff
@@ -3,34 +3,50 @@ import Foundation
 import CoreLocation
 #endif
 
-
-public struct Feature: GeoJSONObject {
-    public var type: FeatureType = .feature
+/**
+ A [Feature object](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2) represents a spatially bounded thing.
+ */
+public struct Feature {
     public var identifier: FeatureIdentifier?
-    public var properties: [String : Any?]?
+    public var properties: JSONObject?
     public var geometry: Geometry
```

Comment:
Spec allows geometry to be null: https://datatracker.ietf.org/doc/html/rfc7946#section-3.2

Not a required change for this PR

Reply:
Oof, I guess we might as well take care of that now rather than later. It’s going to cause some more churn downstream, though any code that uses pattern matching on `geometry` will be unaffected.